test/e2e/hub_templates_encryption_test.go

```diff
@@ -300,6 +300,12 @@ func DebugHubEncryption(policyName, secretName, configMapName, hubNs,
 		"ConfigurationPolicy list": {
 			"get", "configurationpolicy", "-n", clusterNamespace,
 		},
+		"ConfigurationPolicy output": {
+			"get", "configurationpolicy", "-n", clusterNamespace, outYaml,
+		},
+		"Governance-policy-framework-addon log": {
+			"logs", "deployment/governance-policy-framework", "-n", clusterNamespace,
```

Comment:
This is only the correct namespace in hosted mode. Are you able to make it work for both?

Reply:
I see what you mean! Let me fix Thanks

Reply:
I fixed